node/mixins/delim.go

```diff
@@ -31,3 +31,10 @@ func SplitExact(s string, sep string, count int) ([]string, error) {
 	}
 	return ss, nil
 }
+
+// SplitN is an alias of strings.SplitN, which is only present here to
+// make it usable in codegen packages without requiring conditional imports
+// in the generation process.
+func SplitN(s, sep string, n int) []string {
```

Comment:
hmm, just run goimports on the generated code? "figure out what imports we need" is a solved problem in go :)

Reply:
That seems _significantly_ more complex.  I'm not okay with pushing a need for goimports -- something from the `x/tools` repos -- onto users in order to have *compilable* code come out.  (By contrast, while elsewhere we decided we're okay leaning on post-gen invocation of gofmt, that differs on two fronts: it's effectively always available, and it's also not strictly necessary to apply before compile will succeed.)

A small handful of cherry-picked constants like this seems fine to me.

We may want to consider moving such things to their own small package (or the minima file in the output?) to make them less obtrusive, though.

Reply:
I guess goimports is probably also possible to use as a library, so we could just add it to our own dependencies.  But I'd still say for this small of a need, it's probably not worth it.  I'd favor KISS on this until we really have a desire for conditional imports.

Reply:
I was under the assumption we'd be the only ones running this generator. Why would other people generate their own mixins directly?

Reply:
Maybe the comment is unclear -- SplitN is used in the parsing logic of the code that comes *out*.